ortools/linear_solver/xpress_interface_test.cc

```diff
@@ -1253,6 +1253,11 @@ TEST(XpressInterface, setInt64Control) {
   }
 }
 
+TEST(XpressInterface, getInfinity) {
+  MPSolver solver("XPRESS_MIP", MPSolver::XPRESS_MIXED_INTEGER_PROGRAMMING);
+  EXPECT_EQ(std::max(XPRS_PLUSINFINITY, -XPRS_MINUSINFINITY), solver.solver_infinity());
```

Comment:
```suggestion
  EXPECT_LE(solver.solver_infinity(), XPRS_PLUSINFINITY);
  EXPECT_GE(-solver.solver_infinity(), XPRS_MINUSINFINITY);
```

Reply:
actually we want solver.solver_infinity() to be >= XPRESS's definition of infinity, because XPRESS ignores all values >= 1e20 and treats them as infinite
This is why i used the max (because we're not sure it will be XPRS_PLUSINFINITY = -XPRS_MINUSINFINITY always)